firestore/src/android/aggregate_query_android.cc

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "firestore/src/android/aggregate_query_android.h"
+
+#include "firestore/src/android/aggregate_source_android.h"
+#include "firestore/src/jni/compare.h"
+#include "firestore/src/jni/env.h"
+#include "firestore/src/jni/loader.h"
+#include "firestore/src/jni/task.h"
+
+class Query;
```

Comment:
Forward declarations shouldn't generally be used in .cc files. Please remove this forward declaration of the `Query` class. Moreover, this forward declaration should be in the `firebase::firestore` namespace.